ui/shared/components/panel/family/constants.js

```diff
@@ -137,11 +135,11 @@ export const REFERENCE_LOOKUP = ['37', '38'].reduce((acc, genome) => ({
   [genome]: {
     id: GENOME_VERSION_DISPLAY_LOOKUP[GENOME_VERSION_LOOKUP[genome]],
     tracks: REFERENCE_TRACKS.map(({ baseUrl, path, indexPostfix, ...track }) => ({
-      url: `${baseUrl}/${path[genome]}`,
+      url: baseUrl ? `${baseUrl}/${path[genome]}` : path[genome],
```

Comment:
This change is not needed, `baseUrl` is defiend for all the `REFERENCE_TRACKS` its only `REFERENCE_URLS` where that changed